src/features/locators/reducers/changeLocatorElement.thunk.ts

```diff
@@ -57,7 +57,9 @@ export const changeLocatorElement = createAsyncThunk(
       jdnHash: foundHash,
       locator: {
         ...(isCSSLocator ? { cssSelector: locator, xPath: fullXpath } : { xPath: locator, cssSelector }),
-        taskStatus: LocatorTaskStatus.SUCCESS,
+        ...(isCSSLocator
```

Comment:
I think we can avoid this redundant checking and just add `cssSelectorStatus` or `xPathStatus` property with the others on line 59